__tests__/main.test.ts

```diff
@@ -121,3 +129,125 @@ test('does not display module information when it does not exist', async () => {
 
   expect(printChunkModulesTable(statsDiff)).toMatchSnapshot()
 })
+
+describe('printAssetTablesByGroup describes asset sections as requested', () => {
+  // generate all combinations of sections
+  const cases: DescribeAssetsOptions[] = []
+  for (let i = 0; i < Math.pow(2, describeAssetsSections.length); i++) {
+    const options = {} as DescribeAssetsOptions
+    for (let n = 0; n < describeAssetsSections.length; n++) {
+      if ((i >> n) & 1) {
+        options[describeAssetsSections[n]] = true
+      } else {
+        options[describeAssetsSections[n]] = false
+      }
+    }
+    cases.push(options)
+  }
+
+  let statsDiff: WebpackStatsDiff
+  beforeAll(async () => {
+    statsDiff = getStatsDiff(
+      await readJsonFile('./__mocks__/old-stats-assets.json'),
+      await readJsonFile('./__mocks__/new-stats-assets.json')
+    )
+  })
```

Comment:
🎨 Could we get away with just initializing this once? Can't tell right away if there's any mutation going on here or something

```suggestion
  const statsDiff = getStatsDiff(
    await readJsonFile('./__mocks__/old-stats-assets.json'),
    await readJsonFile('./__mocks__/new-stats-assets.json')
  )
```

Reply:
Unfortunately we can't do that since `describe()` can't be async: https://github.com/jestjs/jest/issues/2235

`beforeAll` does indeed only run once before all the test cases within the describe though, unlike `beforeEach`, so it's ugly but should be correct 👍🏽

Reply:
Oh, I see now you are probably wondering if we can change the `let` to a `const`. Not sure of a better way around this but lmk if you think of one!